javascript/action_cable.js

```diff
@@ -1,7 +1,25 @@
 export let consumer
 
+const wait = () => new Promise(resolve => setTimeout(resolve, 1))
```

Comment:
This isn't a change request, just a question... is this a scenario in which there's a meaningful distinction between `1` and `0`? If not, in theory we could get rid of the 2nd argument.

Reply:
didn't try, gimme a sec

Reply:
done!